### PR TITLE
1 splitting widget and app code

### LINKED ISSUE #1: Splitting widget and app code
Copied from https://github.com/house-of-abbey/GarminHomeAssistant/issues/113.

### Goals

1. Having a code base for the app version without "legacy" widget code for achieving space to add new features.
2. Set the widget version to maintenance only mode (feature freeze, only bug fixing etc.)

### Proposal

New repository under "house-of-abbey" for the widget version. For example: "GarminHomeAssistantWidget".

Cloning the current code base into it. Removing all the "legacy" widget related code from the app repo and removing all the glance code in the widget version.

Set the widget repo to "maintenance only mode" by making this clear in the readme.

### PR DESCRIPTION
**Pull request by Someone0nEarth**

### CODE REVIEW

File: source/HomeAssistantApp.mc
Comment (philipabbey) [inline source/HomeAssistantApp.mc:319]: Turns out this comment can be removed now. Prompted me to do the same in the application too.

File: export.cmd
Comment (philipabbey) [inline export.cmd]: I'm going to suggest we add the equivalent for the widget back in. It makes it really easy to build and then deploy the .IQ file.

Widget version based on https://github.com/house-of-abbey/GarminHomeAssistant/blob/113-splitting-widget-and-app-code/export.cmd#L94 i.e. `--output %SRC%\export\HomeAssistant-widget.iq`.

I can do that if you can't test it.

**Review by philipabbey — CHANGES_REQUESTED** [review]
Nice. Thank you for doing this and testing. I'll do a round of beta test deployment too shortly.

File: source/HomeAssistantApp.mc
Comment (Someone0nEarth) [reply source/HomeAssistantApp.mc:319]: Done!

File: export.cmd
Comment (Someone0nEarth) [reply export.cmd]: That would be nice. I don`t have a dev environment under a Windows system.

Comment (philipabbey) [reply export.cmd]: Just testing, then I'll commit the change.

**Comment by philipabbey** [comment]
@Someone0nEarth Just pushed some trivial changes.
HomeAssistantWidget_Savings
Previously this was at 89%, so 5% improvement which is welcome.

**Comment by philipabbey** [comment]
@Someone0nEarth I'll await your approval of these changes before merging.

**Review by philipabbey — APPROVED** [review]

**Comment by philipabbey** [comment]
@Someone0nEarth Are you happy with my few additional changes?

**Comment by Someone0nEarth** [comment]
> @Someone0nEarth Are you happy with my few additional changes?

Was busy the last days. I changed the app name to "Homeassistant Widget" to be able to differentiate the widget and the app version. But I think without "Widget" is working for me too.

**Comment by philipabbey** [comment]
> I changed the app name to "Homeassistant Widget" to be able to differentiate the widget and the app version. But I think without "Widget" is working for me too.

Yes, see screenshot above where it was too wide for the watch face.

**Pull request merged**
